backend/hubs/models/hub.py

```diff
@@ -367,3 +366,94 @@ class Meta:
 
     def __str__(self):
         return "%s" % (self.name)
+
+class HubThemeColor(models.Model): 
+    main = models.CharField(
+        help_text="main color",
+        verbose_name="main color",
+        max_length=200,
+        null=True,
+        blank=True,
+    )
+    light = models.CharField(
+        help_text="light color",
+        verbose_name="light color",
+        max_length=200,
+        null=True,
+        blank=True,
+    )
+    extraLight = models.CharField(
+        help_text="extraLight color",
+        verbose_name="extraLight color",
+        max_length=200,
+        null=True,
+        blank=True,
+    )
+    contrastText = models.CharField(
+        help_text="contrastText color",
+        verbose_name="contrastText color",
+        max_length=200,
+        null=True,
+        blank=True,
+    )
+    background_default = models.CharField(
+        help_text="default background color",
+        verbose_name="default background color",
+        max_length=1024,
+        null=True,
+        blank=True,
+    )
+    background_paper = models.CharField(
+        help_text="paper background color",
+        verbose_name="paper background color",
+        max_length=1024,
+        null=True,
+        blank=True,
+    )
+
+    class Meta:
+        app_label = "hubs"
+        verbose_name = "Hub Theme Color"
+        verbose_name_plural = "Hub Theme Color"
+
+    def __str__(self):
+        return f"ThemeColor {self.id} - Main: {self.main}"
+
+class HubTheme(models.Model): 
```

Comment:
It would be helpful to add a `name` field here which is also shown in django admin. Otherwise it will be really hard to tell them apart when we have many entries here. Name could for example be `Erlangen primary color`.